### ISSUE TITLE
CIPL Flasher

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuuhakuzx/procfw#96]
```
Hi, I wanted to run CIPL Flasher on PSP 3004 4g and it said "This updater does 
not support this model"
Why it is saying this?
It supports only 100x and 200x ?
```

Original issue reported on code.google.com by `mirec.ga...@gmail.com` on 21 Jun 2011 at 9:18

**Comment by GoogleCodeExporter** [comment]
```
yes, only 01g and 02g. 

In fact we can flash CIPL at every model but UNHACKBLE PSP brick.
So the dev of CIPL flasher, neur0n has added it the function which detect the 
model of psp.
According to that, you can avoid brick :)
```

Original comment by `onhe...@gmail.com` on 21 Jun 2011 at 9:45

**Comment by GoogleCodeExporter** [comment]
Original comment by `outma...@gmail.com` on 22 Jun 2011 at 4:27
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
oh....and dont you know when will it be OK for 300x models?
```

Original comment by `mirec.ga...@gmail.com` on 22 Jun 2011 at 2:10

**Comment by GoogleCodeExporter** [comment]
```
its is only a few seconds waiting.
i am happy. :)

Ps. On the site www.psp-media.nl 
1 BRICK with 6.20 p fixed with the repair sevice sony.
1 semi-brick with 6.20 p (my psp was that) fixed with update to 6.39

so wy permanent, it is good enough.

sorry for my bad english. 

```

Original comment by `vanmella...@gmail.com` on 22 Jun 2011 at 5:39

**Comment by GoogleCodeExporter** [comment]
```
i hope it won't take so long :/ i hope it won't take longer than 1 week....
```

Original comment by `mirec.ga...@gmail.com` on 22 Jun 2011 at 7:17
